src/dfcx_scrapi/core/intents.py

```diff
@@ -68,11 +68,11 @@ def intent_proto_to_dataframe(
         """intents to dataframe
 
         Args:
-          obj, intent protobuf object
-          mode: (Optional) basic returns display name and training phrase as
-            plain text.
-          Advanced returns training phrase and parameters df broken out by
-            parts.
+          obj (Intent protobuf)
```

Comment:
@MRyderOC for documentation here, you can omit the types inside the docstring. The type is already provide in the function definition. This is how the remainder of the library is currently documented, so this provides consistency throughout the docs.

For example:
```
obj: Intent Protobuf
mode: The operational mode of the function which determines the output
  "basic" returns display name and training phrase as plain text.
  "advanced" returns training phrases broken out by parts with their parameters included.
```